runtime/balrt.h

```diff
@@ -447,7 +460,7 @@ typedef enum {
     FILL_COPY
 } Fillability;
 
-extern Fillability _bal_structure_create_filler(MemberType memberType, StructureDescPtr fillerDesc, TaggedPtr *valuePtr);
+TaggedPtr structCreateFiller(FillerDescPtr fdp, Fillability *fillability);
```

Comment:
Make this static inline and move downwards

Reply:
fixed